src/django/pfb_analysis/fixtures/analysis-score-metadata.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "model": "pfb_analysis.AnalysisScoreMetadata",
```

Comment:
@spencerrecneps this is the file you'll need to update once you're ready to finalize the score metadata. I just filled in most fields with placeholders for now.

Opened #235 as a reminder.